Microsoft.HealthVault.Fhir/Constants/HealthVaultVocabularies.cs

```diff
@@ -22,21 +22,34 @@ public static class HealthVaultVocabularies
         public const string VitalStatistics = "vital-statistics";
         public const string BloodGlucoseMeasurementContext = "glucose-measurement-context";
         public const string BloodGlucoseMeasurementType = "glucose-measurement-type";
-        public const string OutsideOperatingTemperature = "outside-operating-temperature";
-        public const string ReadingNormalcy = "reading-normalcy";
-        public const string IsControlTest = "is-control-test";
-
+        
         public const string Fhir = "fhir";
         public const string Wc = "wc";
 
-        public const string StateFhirExtensionName = "http://healthvault.com/fhir-extensions/thing-state";
-        public const string FlagsFhirExtensionName = "http://healthvault.com/fhir-extensions/thing-flags";
+        public const string BaseUri = "http://healthvault.com/";
+        public const string StateFhirExtensionName = BaseUri + "fhir-extensions/thing-state";
+        public const string FlagsFhirExtensionName = BaseUri + "fhir-extensions/thing-flags";
+
+        public const string OutsideOperatingTemperatureExtensionName = BaseUri + "blood-glucose/outside-operating-temperature";
```

Comment:
For most of these, couldn't we create a helper that does this? Basically we have a const for the BaseUri, but it then just takes the object type name and property name. Something like:
`$"{baseUri}{objectName.ToLower()}/{propertyname.ToLower()}/"`